hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -612,6 +623,21 @@ public void initializeBootstrapDirsIfNotExists() throws IOException {
     initializeBootstrapDirsIfNotExists(getHadoopConf(), basePath.toString(), getFs());
   }
 
+  private static HoodieTableMetaClient newMetaClient(Configuration conf, String basePath, boolean loadActiveTimelineOnLoad,
+      ConsistencyGuardConfig consistencyGuardConfig, Option<TimelineLayoutVersion> layoutVersion,
+      String payloadClassName, FileSystemRetryConfig fileSystemRetryConfig, Properties props) {
+    HoodieMetastoreConfig metastoreConfig = null == props
+        ? new HoodieMetastoreConfig.Builder().build()
+        : new HoodieMetastoreConfig.Builder().fromProperties(props).build();
+    return metastoreConfig.enableMetastore()
+        ? (HoodieTableMetaClient) ReflectionUtils.loadClass("org.apache.hudi.common.table.HoodieTableMetastoreClient",
```

Comment:
Add some instructions to remove this reflection after the HoodieTableMetastoreClient is fully integrated.

Reply:
HoodieTableMetastoreClient is under `hudi-metastore`, so it has to use reflection here

Reply:
> hudi-metastore

after `hudi-metastore` ready, can remove this reflection. add some note for this code?

Reply:
hudi-metastore depends on the hudi-common, reflection is necessary.

Reply:
> hudi-metastore depends on the hudi-common, reflection is necessary.

well, ok